apps/mull-ui/src/app/app.tsx

```diff
@@ -13,6 +15,14 @@ import './app.scss';
 /**
  * Main component of the application
  */
+const showHeader = () => {
+  const location = useLocation();
+  if (['/login', '/register'].includes(location.pathname)) {
```

Comment:
Please change this line to
```suggestion
  if ([ROUTES.LOGIN, ROUTES.REGISTER].includes(location.pathname)) {
```

and change the routes enum to an object:

```ts
export const ROUTES = {
  HOME: '/home',
  MAP: '/map',
  CREATE_EVENT: '/create-event',
  TOOLS: '/tools',
  MESSAGES: '/messages',
  PROFILE: '/profile',
  LOGIN: '/login',
  REGISTER: '/register',
};
```

I tested it and thankfully nothing breaks when you do that

Reply:
Great catch cristian, will do 👍